src/browser-connection/gateway.js

```diff
@@ -36,7 +37,8 @@ export default class BrowserConnectionGateway {
         this._dispatch('/browser/status/{id}', proxy, BrowserConnectionGateway.onStatusRequest);
 
         proxy.GET('/browser/assets/index.js', { content: IDLE_PAGE_SCRIPT, contentType: 'application/x-javascript' });
-        proxy.GET('/browser/assets/style.css', { content: IDLE_PAGE_STYLE, contentType: 'text/css' });
+        proxy.GET('/browser/assets/styles.css', { content: IDLE_PAGE_STYLE, contentType: 'text/css' });
+        proxy.GET('/favicon.png', { content: FAVICON, contentType: 'image/x-icon' });
```

Comment:
It would be nice to remove this condition from hammerhead: https://github.com/inikulin/testcafe-hammerhead/blob/master/src/proxy/index.js#L118 to enable serving of the favicon. And register `favicon.ico` (not png!) here: https://github.com/superroma/testcafe-phoenix/blob/master/src/index.js#L28 So, favicon will automatically will be served for all pages (including tested), not only browser idle page